### PR TITLE
Update styleguide branding.

### PR DESCRIPTION
**Pull request by sarahmonster**
This PR is mostly going to be about making the styleguide itself pretty, because that's how I roll. Styling will wind up being something similar to the Sketch mockups:

1  Principles

**However!** Before I get too deep in the woods doing that, I'd love to know if this structure seems good to build on top of. Notabaly:

- Does the overall component structure make sense? 
- Does the variable structure seem like it will work for our purposes? (Variables are in src/base right now, but I'll likely shift them into CSS files instead.)
- Does Styleguideist seem like a workable solution for the style guide itself? (I have mixed feelings, but that's just because their documentation is a little hairy.)
- Will this be easily published as a package and consumed by other applications? (Right now I'll be doing it locally via `npm link`, but we'll want a better long-term solution.)
- Anything else we should consider before barging ahead?

### CODE REVIEW

**Comment by tofumatt** [comment]
I have one comment on the text in the design principles, and it's related to the "Be Bold" section:

> It's better to be loathed than indifferent.

Two things:

1. "to be loathed" is a verb but "to be indifferent" isn't. That's a bit syntactically awkward.
2. "to be loved or loathed" makes more sense because there should be a benefit to the few that loathe. The aim here is to make something people, hopefully _more_ people, love and connect with whilst invariably turning off a few because of the bolder decisions made. It might help to communicate that.

I have more comments to come but that's a thought.

**Comment by sarahmonster** [comment]
Yes, good points! The text in the design principles is still pretty drafty, but once I move them into the design system proper they'll get a copyedit, a bit more thought, and proper review! I was mostly writing quick text to fill the boxes. 😜 

For right now, I'm looking more for feedback on the technical architecture itself before I go ahead and start styling things and inputting actual content. I just don't want to do a lot of work on this specific structure if it's not going to work for our purposes.

**Comment by tofumatt** [comment]
Ten-four, it was just fresh in my brain and wanted to share the feedback. Will technical-review next.

**Comment by sarahmonster** [comment]
It's a good comment! Since these are pretty important I'll open them as a separate PR (along with other elements as well) so we can be sure they get our full consideration. 

Do let me know if I can help make it easier to review this setup! I can revert everything and submit it all as a PR if that would make sense as well.

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
-  "name": "ether-system-react",
+  "name": "nautilus-design-system-react",
```
Comment (tofumatt) [inline package.json]: This is the name it's going to appear as in npm, so just make sure it's what you want 😄

File: styleguide.config.js [hunk 2]
```diff
@@ -13,19 +13,23 @@ module.exports = {
       ],
     },
   },
+  theme: {
+    color: {
+      link: 'firebrick',
+      linkHover: 'hotpink',
```
Comment (tofumatt) [inline styleguide.config.js]: 👍

File: package.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
...
   "version": "1.0.3",
```
Comment (tofumatt) [inline package.json]: I'd knock this waaaaaay down. We're like… `0.1.0`.

File: package.json (hunk 1 above)
Comment (sarahmonster) [reply package.json]: Pretty sure this changes in the next PR—I'm using `nautilus-system` right now, but  I'm open to something more elegant. `nautilus-react` maybe?

File: styleguide.config.js (hunk 2 above)
Comment (sarahmonster) [reply styleguide.config.js]: Not permanent!

File: package.json (hunk 1 above)
Comment (tofumatt) [reply package.json]: `nautilus-react` works for me. 👍🏻

**Pull request merged**